auditbeat/docs/breaking.asciidoc

```diff
@@ -0,0 +1,129 @@
+[[auditbeat-breaking-changes]]
```

Comment:
Might be a good idea to add something to the [breaking changes](https://www.elastic.co/guide/en/beats/libbeat/current/breaking-changes-6.0.html) section of the platform reference that points to the Auditbeat topic.

Reply:
I think that it would be good to add something over there, but that page is specific to 6.0. Perhaps on the [parent page](https://www.elastic.co/guide/en/beats/libbeat/current/breaking-changes.html) to that one we could add a list of links?

Reply:
Yes, a list of links might work, or we could add a container for 6.2 that just points to the Auditbeat topic. If Auditbeat is the only Beat that has breaking changes, a container for 6.2 is overkill, IMO.